### ISSUE TITLE
An error occurred if the type takes string parameters

### ISSUE THREAD

**Issue by ybrliiu** [kfly8/p5-Function-Return#27]
An error occurred if the type takes string parameters, like this.

```perl
use v5.36;
use warnings;
use utf8;
use Types::Standard -types;
use Function::Return;

sub hoge :Return(Enum['red', 'blue', 'green']) {
  'red';
}
```

```
Bareword found where operator expected at (eval 125) line 2, near ""Too few return values for fun hoge (expected Enum["red"
	(Missing operator before red?)
String found where operator expected at (eval 125) line 2, near "red",""
Bareword found where operator expected at (eval 125) line 2, near "","blue"
	(Missing operator before blue?)
String found where operator expected at (eval 125) line 2, near "blue",""
Bareword found where operator expected at (eval 125) line 2, near "","green"
	(Missing operator before green?)
String found where operator expected at (eval 125) line 2, near "green"], got @{[map { defined $_ ? $_ : 'undef' } @ret]})""
Bareword found where operator expected at (eval 125) line 3, near ""Too many return values for fun hoge (expected Enum["red"
	(Missing operator before red?)
String found where operator expected at (eval 125) line 3, near "red",""
Bareword found where operator expected at (eval 125) line 3, near "","blue"
	(Missing operator before blue?)
String found where operator expected at (eval 125) line 3, near "blue",""
Bareword found where operator expected at (eval 125) line 3, near "","green"
	(Missing operator before green?)
String found where operator expected at (eval 125) line 3, near "green"], got @{[map { defined $_ ? $_ : 'undef' } @ret]})""
apply_subname requires subroutine reference at lib/Function/Return.pm line 166.
Execution of tmp.pl aborted due to compilation errors.
```

The reason is that the type name contains `"`, which causes the `"` to be doubled when evaling.
https://github.com/kfly8/p5-Function-Return/blob/5e95a28eb5803ad571f312d360ff3ce73441d3e2/lib/Function/Return.pm#L113-L117

**Comment by ybrliiu** [comment]
I fixed it #28 .

**Comment by kfly8** [comment]
Thanks!
